internal/pkg/service/buffer/api/service/service.go

```diff
@@ -159,67 +163,25 @@ func (s *service) UpdateReceiver(dependencies.ForProjectRequest, *UpdateReceiver
 func (s *service) GetReceiver(d dependencies.ForProjectRequest, payload *GetReceiverPayload) (res *Receiver, err error) {
 	ctx, str := d.RequestCtx(), d.Store()
 
-	projectID, receiverID := d.ProjectID(), payload.ReceiverID
-
-	receiver, err := str.GetReceiver(ctx, projectID, string(receiverID))
+	receiverKey := key.ReceiverKey{ProjectID: d.ProjectID(), ReceiverID: string(payload.ReceiverID)}
+	receiver, err := str.GetReceiver(ctx, receiverKey)
 	if err != nil {
 		return nil, err
 	}
 
-	exportList, err := str.ListExports(ctx, projectID, string(receiverID))
+	exportList, err := str.ListExports(ctx, receiverKey)
 	if err != nil {
 		return nil, err
 	}
 
-	exports := make([]*Export, 0, len(exportList))
-	for _, export := range exportList {
-		mapping, err := str.GetMapping(ctx, projectID, string(receiverID), export.ID)
```

Comment:
Toto mi hlasil linter ako duplicitny kod, hodil som to do funkcie (docasne, kym bude nejaky ten mapper).

Reply:
Na tom mapperu už dělám